app/src/main/java/com/android/calendar/settings/GeneralPreferences.kt

```diff
@@ -231,7 +231,7 @@ class GeneralPreferences : PreferenceFragmentCompat(),
 
     override fun onStart() {
         super.onStart()
-        preferenceScreen.sharedPreferences.registerOnSharedPreferenceChangeListener(this)
+        preferenceScreen.sharedPreferences?.registerOnSharedPreferenceChangeListener(this)
```

Comment:
I'm fine with updating to 1.2.0, could you move all the related changes into an extra commit?
Otherwise we could do it in a separate PR to get this merged already.